apps/sched/interface.html

```diff
@@ -185,15 +285,26 @@ <h4>Manage dated events</h4>
     <div class="float-right">
       <button class="btn" onclick="addAlarm()">
         <i class="icon icon-plus"></i>
+        Alarm (with date)
```

Comment:
This is named "Event" in the watch.
I think we should have the same name for it.

Reply:
Thank you - I was struggling to come up with a name to differentiate, as you can perhaps tell